src/dosbox.cpp

```diff
@@ -815,11 +815,12 @@ void DOSBOX_Init()
 	pint = secprop->Add_int("voodoo_perf", only_at_start, 1);
 	pint->SetMinMax(0, 3);
 	pint->Set_help("Performance optimisations to use when emulating the 3dfx Voodoo card:\n"
-	               "Note: emulation is software-based and does not use host-level OpenGL calls.\n"
 	               "   0:  No optimizations.\n"
 	               "   1:  Multi-threading (default).\n"
 	               "   2:  Disable bilinear filtering.\n"
-	               "   3:  All optimizations (both 1 and 2).");
+	               "   3:  All optimizations (both 1 and 2).\n"
+	               "Notes: Voodo emulation is software-based and does not use host-level\n"
```

Comment:
It's a single note, so `Note:`?

Reply:
We use `Notes:` everywhere.

Reply:
Fair enough. If it's the same everywhere else, who am I to argue?

Reply:
Ok, on second thought, you're right @shermp . I'll refactor all existing descriptions in an additional commit so it's consistent:
- For singles note, it's `Note`
- For multiple notes in a bulleted list, it's `Notes`

Thanks for pointing this out! 🎉